cache-config/testing/docker/traffic_ops/Dockerfile

```diff
@@ -20,19 +20,23 @@
 # Based on CentOS 8
 ############################################################
 
-ARG RHEL_VERSION=8
-FROM centos:${RHEL_VERSION}
-ARG RHEL_VERSION=8
-# Makes RHEL_VERSION available in later layers without needing to specify it again
-ENV RHEL_VERSION=$RHEL_VERSION
+ARG OS_VERSION=8
+ARG OS_DISTRO=rockylinux/rockylinux
```

Comment:
Rocky Linux now has an official image, `rockylinux/rockylinux` can be just `rockylinux`.